lib/mix/tasks/format.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Mix.Tasks.Format do
+  use Mix.Task
+
+  alias ExFormat
```

Comment:
There is no need for this, it's not doing anythign :) `alias` only does something when there is a `.` in the module name or when you pass `:as` as an option.